keepercommander/loginv3.py

```diff
@@ -110,20 +111,29 @@ def login(params: KeeperParams):
                 raise Exception('This account need to be created.' % rest_api.CLIENT_VERSION)
 
             elif resp.loginState == proto.REGION_REDIRECT:
+                p = urlparse(params.server)
+                new_server = urlunparse((p.scheme, resp.stateSpecificValue, '/api/rest/', None, None, None))
```

Comment:
We don't really need `/api/rest/` in the url. config.json can have just `https://keepersecurity.[com|eu|com.eu]`